src/UI/class-plugins-actions.php

```diff
@@ -39,13 +39,25 @@ public function run(): void {
 	 * @return array
 	 */
 	public function add_plugin_meta_links( array $actions ): array {
-		$settings_link = sprintf(
+		if ( is_multisite() && is_plugin_active_for_network( plugin_basename( PARSELY_FILE ) ) ) {
+			$actions['siteslist'] = sprintf(
+				'<a href="%s" target="_blank">%s</a>',
```

Comment:
Do existing links have a target attribute? It has accessibility implications, and I'd rather we let folks choose to open it in a new tab if they wish, instead of forcing it.